src/index.html

```diff
@@ -1,17 +1,210 @@
 <!doctype html>
 <html lang="en">
+
   <head>
     <meta charset="UTF-8">
-    <meta
-      name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+    <meta name="viewport"
+    content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
     >
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>HTML Form</title>
     <link rel="stylesheet" href="./style.css">
   </head>
+
   <body>
-    <h1>HTML Form</h1>
+
```

Comment:
You shouldn't have empty lines between parent and child elements, generally make empty lines between siblings like in this example:
```html
<div>
  <span>First child</span>

  <p>Second child</p>
</div>
```
There are exemptions from this rule, for example when you have repeated elements like in this example:
```html
<ul>
  <li></li>
  <li></li>
  <li></li>
  <li></li>
</ul>
```

Reply:
Keep this in mind and adjust rest of your code